### PR TITLE
[WIP] feat(footnotes): make footnotes small in Learn

### PR DESCRIPTION
**Pull request by nimbinatus**
For the Learn pathways, we want to move the glossary links. I was thinking a footnote works well, but it looks odd at the end. Made them smaller in the css for now.

### CODE REVIEW

File: src/scss/_learn.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.footnotes {
+    font-size: 0.75rem;
```
Comment (susanev) [inline src/scss/_learn.scss:2]: i dont totally know whats going on but can we not use tailwind for this?
`text-sm` or maybe xxs?

Comment (nimbinatus) [reply src/scss/_learn.scss:2]: I tried. Unfortunately, the markdown processor renders footnotes after all the classes are applied. The only way I can affect the footnotes is to change the behavior of the default class it applies like this. I've tried stuff like adding `markdown=1` to the HTML, wrapping it in other tags to apply the other classes... Goldmark doesn't have a class-apply function for non-headings, from what I've been able to find.

**Comment by nimbinatus** [comment]
Update: After discussion, holding this one to review as the footnote arrows that are auto-generated look pretty similar to other arrows on the page and can confuse the user.

**Comment by susanev** [comment]
@nimbinatus could we close this out? or would you prefer to keep it around?

**Comment by nimbinatus** [comment]
I could open footnotes as an issue and we could close this one if that's easier. I don't need this one open for any specific reason as long as the need is documented.

**Comment by susanev** [comment]
@nimbinatus id super appreciate you opening up an issue instead!

**Pull request closed without merging**